ios/scripts/fixup-frameworks.sh

```diff
@@ -36,10 +36,4 @@ do
         rm "$FRAMEWORK_EXECUTABLE_PATH"
         mv "$FRAMEWORK_EXECUTABLE_PATH-merged" "$FRAMEWORK_EXECUTABLE_PATH"
     fi
-
-    echo "Code signing framework"
```

Comment:
@lyubomir FYI: This is no longer needed as embedding the framework signs it already. Before we had to do this by hand because there is no "embed framework" build step for a framework.